ITE/4/README.adoc

```diff
@@ -430,35 +432,76 @@ resulting build and CI report.
 ==== Use Case 3: Granular JSON Access
 
 The enhancement proposed in this ITE can also be used to provide more granular
-access to certain resources, such as JSON files. A generic URI can be used to
-resolve to the information contained in a specific key of a JSON file. This allows
-for greater flexibility when using in-toto with different types of artifacts.
+access to resources such as JSON files. A generic URI can be used to resolve to
+the information contained in a specific key of a JSON file using JSONPath. This
+allows for greater flexibility when using in-toto with different types of
+artifacts.
+
+A practical example of this use case is for Cloud Native Application Bundles
+(CNAB) metadata. A link:https://github.com/cnabio/cnab-spec/blob/master/101-bundle-json.md[CNAB bundle.json file]
+contains information such as the top-level package information, images included
+in the bundle, credentials required by the application, and so on. We define a
+minimal supply chain described below for the creation of these bundle.json
+files. A link:https://github.com/adityasaky/in-toto/tree/ite-4-cnab-resolver[prototype resolver]
+was created using the structure defined in the <<uri-resolvers, URI Resolvers>>
+section for CNAB bundle.json files.
+
+===== in-toto link attestation for the creation of the initial bundle.json
+
+In this step of the minimal supply chain, a developer creates an initial
```

Comment:
Maybe link to this example root layout for clarity?

https://github.com/cnabio/signy/pull/74

Reply:
Do you think it's better to point to it after it's reviewed and merged?

Reply:
I think it's okay to point to PR, which at least makes it clear where the relevant code is...